### PR TITLE
Cleanup build options

### PR DESCRIPTION
**Pull request by jasonLaster**
* switch to using an implicit bundleName over just using "bundle" or passing in an explicit one
* drop library name 
* cleanup some code

### CODE REVIEW

File: packages/devtools-launchpad/bin/development-server.js [hunk 1]
```diff
@@ -35,18 +35,26 @@ function httpOrHttpsGet(url, onResponse) {
...
+
 function serveRoot(req, res) {
   const tplPath = path.join(__dirname, "../index.html");
   const tplFile = fs.readFileSync(tplPath, "utf8");
 
-  let favicon = getValue("favicon");
   res.send(Mustache.render(tplFile, {
     isDevelopment: isDevelopment(),
     dir: getValue("dir") || "ltr",
+    bundleName: getValue("title").toLocaleLowerCase() || "bundle",
```
Comment (nchevobbe) [inline packages/devtools-launchpad/bin/development-server.js]: wouldn't it break if the config does not specify a title ?

Comment (wldcordeiro) [inline packages/devtools-launchpad/bin/development-server.js]: @nchevobbe The or should fall back to bundle.

Comment (nchevobbe) [inline packages/devtools-launchpad/bin/development-server.js]: I was thinking `getValue("title")` is undefined if there isn't a title entry in the configuration, hence, doing `.toLocaleLowerCase()` on undefined would have throw.
I'll check and open a PR if there's such issue

**Comment by jasonLaster** [comment]
sorry if i missed something @nchevobbe

**Comment by nchevobbe** [comment]
> sorry if i missed something @nchevobbe

nope, sounds good, I didn't saw that you fixed it :)

**Pull request merged**